### ISSUE TITLE
[Feature Request] Add to playlist

### ISSUE THREAD

**Issue by rattrapper** [elbowz/xbmc.service.pushbullet#26]
Would be cool to enable new option for handling video, audio and image pushes - enqueue (add to current playlist)

**Comment by ruuk** [comment]
Good idea. Hopefully I get a chance to work on this addon again in the near future :)
